tests/openQA/dashboard.pm

```diff
@@ -8,8 +8,11 @@ sub run {
     prepare_firefox_autoconfig;
     switch_to_x11;
     ensure_unlocked_desktop();
+    start_gui_program('firefox http://localhost', 60, valid => 1);
+    assert_screen 'openqa-dashboard', 60;
```

Comment:
Are the delays here fine? I reckon this could be optimized further.

Reply:
Note that this is 60s max in the worst case, not a delay.

Reply:
I suggest to not even do those additional checks here by starting firefox as that is rather expensive. Just revert the code to check for localhost. If that works then we can be sure that also 127.0.0.1 and [::1] works. We should focus the openQA-in-openQA tests to test usual use cases, not cover all corner cases.